### PR TITLE
slack webhook integration

### PR DESCRIPTION
**Pull request by zpeyton**
For this to work in prod you'll need a SLACKWEBHOOKBOTCHANNEL env var in prod lambda config. (Liv updated the lambda so we're good)

### CODE REVIEW

**Review by j256 — APPROVED** [review]
hard to vet without trying it out but it looks well formed.

File: lib/slack.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+const https = require("https");
+
+const slackWebhookHost = "hooks.slack.com";
+
+const channelConfig = {
```
Comment (livgust) [inline lib/slack.js:5]: I set this in lambda so it's always SLACKWEBHOOKDEVCHANNEL, it's just set to the appropriate value depending on context

**Review by livgust — CHANGES_REQUESTED** [review]

File: lib/RxTouch.js [hunk 2]
```diff
@@ -157,6 +158,7 @@ async function ScrapeRxTouch(browser, site, siteName) {
             }
             if (!process.env.DEVELOPMENT) {
                 await s3.savePageContent(`${siteName}-${loc.zip}`, page);
+                await sendSlackMsg("dev", "Found appointments on ${siteName}");
```
[the remark this answers is not in the document]
Comment (zpeyton) [reply lib/RxTouch.js]: I removed that from the commit anyway. You are right that we don't need `await`

File: lib/slack.js (hunk 1 above)
Comment (zpeyton) [reply lib/slack.js:5]: Per our convo, I think we're all set here. SLACKWEBHOOKBOTCHANNEL is correct SLACKWEBHOOKBOTTESTINGCHANNEL is just for DEVELOPMENT.

**Comment by tkjones117** [comment]
I tried adding this to a scraper (along with editing my .env file):
`await slackBot.sendSlackMsg("bot", "Test message from Tessa, for PR review purposes!");`

Got error:
`Error status code [302] returned from schedule request: Found`

File: lib/RxTouch.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (zpeyton) [reply lib/RxTouch.js]: That code was removed from the PR in general and we probably do not need await here but there could be a situation where the scraper finishes before the request has a chance to finish. I needed it with a return following that statement.

[the remark this answers is not in the document]
Comment (johnhawkinson) [reply lib/RxTouch.js]: If it does ever come back, perhaps `Promise.all()`

**Review by livgust — APPROVED** [review]

**Pull request merged**